sources/Cargo.toml

```diff
@@ -226,7 +226,7 @@ version = "0.1.0"
 
 [workspace.dependencies.settings-extension-oci-defaults]
 git = "https://github.com/bottlerocket-os/bottlerocket-settings-sdk"
-tag = "bottlerocket-settings-models-v0.11.0"
+tag = "bottlerocket-settings-models-v0.12.0"
```

Comment:
I wonder why we didn't explicitly refer to settings-extension and its subpackages like `settings-extension-container-runtime` here. 

And if we did, those would need to be bumped. But I confirmed that from the Cargo.lock introduced in this PR that those versions are correctly updated. Approving.

Reply:
It looks like we actually depend on the oci-defaults here: https://github.com/bottlerocket-os/bottlerocket-core-kit/blob/4bb9a8ad831aec5018ec8903f8377de942326387/sources/api/schnauzer/Cargo.toml#L45